devicemodel/hw/vga.c

```diff
@@ -339,7 +338,9 @@ vga_mem_rd_handler(struct vmctx *ctx, uint64_t addr, void *arg1)
 		/*
 		 * monochrome text mode: base 0xb0000 size 32kb
 		 */
-		assert(0);
+		offset -=0xb0000;
```

Comment:
We need to check if the offset exceed the 0xb0000.

Reply:
Will fix in next version.